### PR TITLE
Laravel 7 and autoload blade custom directives

### PR DESCRIPTION
**Pull request by jpscharf**
- Updated for Laravel 7
- Can now autoload Balde custom directives:
  - Added an option to the config to autoload custom directives.
  - Added env variables to config.
  - Updated README.
  - Deferred StringBladeServiceProvider.
  - Added `vendor/` to .gitignore.

### CODE REVIEW

File: config/blade.php [hunk 1]
```diff
@@ -4,6 +4,8 @@
     // How many seconds past compiled file last modified time to recompile the template
     // A value of 0 is always recompile
     // Note: homestead time verses pc time may be off
-    'secondsTemplateCacheExpires' => 300
+    'secondsTemplateCacheExpires' =>  env('STRING_BLADE_CACHE_TIMEOUT') ?? 300,
 
+    // Determine whether the service provider to autoload blade custom directives.
+    'autoload_custom_directives' => env('STRING_BLADE_AUTOLOAD') ?? false,
```
Comment (veneliniliev) [inline config/blade.php]: env('STRING_BLADE_AUTOLOAD', false)

File: config/blade.php [hunk 2]
```diff
@@ -4,6 +4,8 @@
     // How many seconds past compiled file last modified time to recompile the template
     // A value of 0 is always recompile
     // Note: homestead time verses pc time may be off
-    'secondsTemplateCacheExpires' => 300
+    'secondsTemplateCacheExpires' =>  env('STRING_BLADE_CACHE_TIMEOUT') ?? 300,
```
Comment (veneliniliev) [inline config/blade.php]: env('STRING_BLADE_CACHE_TIMEOUT', 300) ?

Comment (jpscharf) [reply config/blade.php]: I changed it.

File: config/blade.php (hunk 1 above)
Comment (jpscharf) [reply config/blade.php]: I changed it.

**Comment by veneliniliev** [comment]
@TerrePorter please merge this :)

**Comment by abishekrsrikaanth** [comment]
@TerrePorter, any chance you can review this and merge.

**Pull request merged**